packages/portfolio-contract/src/resolver.exo.ts

```diff
@@ -0,0 +1,269 @@
+/**
+ * @file CCTP Transaction Resolver - handles cross-chain transaction confirmations
```

Comment:
I expect the resolver will handle more than CCTP. So:

```suggestion
 * @file Transaction Resolver - handles cross-chain transaction confirmations
```

Otherwise, change the name to `cctp-resolver.exo.ts`.